CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
1. Creating CMake packages (https://cmake.org/cmake/help/v3.8/manual/cmake-packages.7.html) is not for Unix-like systems only. See also ExternalPackage module and Hunter package manager.
2. The INSTALL step in CMake is required to create the CMake package because only the CMake files describing the package exported in the INSTALL step are relocatable and can be imported by other projects.
Maybe using `make install` equivalent on Windows to install files to `C:\Program Files` does not make much sense, but using this to build dependencies locally definitely does.
3. GNUInstallDirs has good defaults that work on Windows as well. Defaults are `bin`, `lib`, etc. GNUInstallDirs just solved issue on systems where libs should be installed to `lib64` instead of `lib` dir.

@ruslo, do you want to add something about CMake packages and Hunter?